atomicapp/nulecule/base.py

```diff
@@ -174,11 +190,25 @@ def load_from_path(cls, src, config=None, namespace=GLOBAL_CONF,
 
             raise NuleculeException("Failure parsing %s file. Validation error on line %s, column %s:\n%s"
                                     % (nulecule_path, line, column, output))
-
-        nulecule = Nulecule(config=config, basepath=src,
-                            namespace=namespace, **nulecule_data)
-        nulecule.load_components(nodeps, dryrun)
-        return nulecule
+        else:
+            kwargs = deepcopy(nulecule_data)
+            kwargs['progenitor'] = True
+            del kwargs[GRAPH_KEY]
+            progenitor = Nulecule(config=config, basepath=src,
```

Comment:
Isn't ``progenitor`` always going to be ``None`` since ``__init__()`` of ``Nulecule`` does not return anything.